src/middlewares/Authenticate.js

```diff
@@ -0,0 +1,58 @@
+import { verifyToken, errorResponse } from '../utils';
+import models from '../models';
+
+const { Profiles, Sequelize } = models;
+/**
+ * @class Authenticate
+ * @description authenticate tokens and roles
+ * @exports Authenticate
+ */
+class Authenticate {
+  /**
+   * Verify if token is valid
+   * @param  {object} req - The user request object
+   * @param  {object} res - The user res response object
+   * @param  {function} next - The next() Function
+   * @returns {String} req.userId - The user id
+   */
+  static async verifyToken(req, res, next) {
+    try {
+      const { headers: { authorization } } = req;
+      const token = authorization.split(' ')[1];
+      const decoded = await verifyToken(token);
+      if (!decoded) {
+        return errorResponse(res, 401, 'Access denied we could not verify user');
```

Comment:
Correct this sentence. There should be a period after denied. E.g. `Access denied. We could not verify your identity`

Reply:
Done